### PR TITLE
fix: small WFE bugs [BIVS-2050]

### PR DESCRIPTION
**Pull request by morbalint**
there was an issue with `undefined !== null` when checking for missing value, the fix is using `!=` instead.

### CODE REVIEW

File: source/javascripts/services/_appService.js.erb [hunk 1]
```diff
@@ -424,11 +424,11 @@
 					}
 
 					appService.validationWarnings = [];
-					if (validationData.warnings.config !== null) {
+					if (validationData.warnings.config.length != null) {
```
Comment (roland-spekker-bitrise) [inline source/javascripts/services/_appService.js.erb:427]: The previous version checks if the config isn't null and now there is no null check and we check if the length isn't null? The length of an array won't be null.

**Review by roland-spekker-bitrise — CHANGES_REQUESTED** [review]
Can you elaborate on this fix?

File: source/javascripts/services/_appService.js.erb (hunk 1 above)
Comment (morbalint) [reply source/javascripts/services/_appService.js.erb]: nope, fixed

**Comment by morbalint** [comment]
> Can you elaborate on this fix?

added a more detailed description

**Review by roland-spekker-bitrise — APPROVED** [review]

**Pull request merged**